### PR TITLE
changed test for empty json in getContentAsDataFrame()

### PR DESCRIPTION
**Pull request by geneorama**
oops, forgot that we use "issue" instead of "iss" for the branch names.  

Anyway, pretty straightforward fix.  I think the problem might have been introduced by Socrata, or might have been introduced by a change in `httr`, or maybe Kevin just found something new!

### CODE REVIEW

**Comment by geneorama** [comment]
added test for the example url

File: DESCRIPTION [hunk 1]
```diff
@@ -10,7 +10,7 @@ Description: Provides easier interaction with
     format and manages throttling by 'Socrata'.
     Users can upload data to Socrata portals directly
     from R.
-Version: 1.7.1-12
+Version: 1.7.1-13
 Date: 2016-03-13
```
Comment (tomschenkjr) [inline DESCRIPTION]: Can you also update `Date` as well?

File: tests/testthat/test-all.R [hunk 2]
```diff
@@ -356,3 +356,26 @@ test_that("fully replace a dataset", {
...
+  #             "cleared_time,census_tract_2010_geoid",
+  #             "&$where=incident_date=%272016-08-21%27")
+  
+  ## This request has been modified to avoid the rbind issue
+  u <- paste0("https://data.smgov.net/resource/xx64-wi4x.json?$",
+              "select=incident_number,incident_date,call_type,received_time,",
+              "cleared_time,census_tract_2010_geoid",
+              "&$where=incident_date=%272016-08-27%27%20and%20",
+              "census_tract_2010_geoid%20is%20not%20null")
+  library(RSocrata)
```
Comment (tomschenkjr) [inline tests/testthat/test-all.R]: Shouldn't need to `library(RSocrata)`, that is done at the top of the `/tests/testthat.R` file.

**Comment by tomschenkjr** [comment]
@geneorama - I'm reruning the CI builds, but looks like it's just a conflict on the uploading process.

I made a couple of small comments in the code. Can you take a look at those and then also update this pull request to `dev` branch? I'll also merge #94 into `dev` for a test before pushing to `master`.

**Comment by geneorama** [comment]
I didn't mean to include the`library(RSocrata)`, obviously had that in there while testing elsewhere.  Updated the date.
Thanks for catching.

**Comment by tomschenkjr** [comment]
@geneorama - one more fix: can you adjust the target branch to `dev`?

**Comment by geneorama** [comment]
@tomschenkjr Yes, meant to use `dev` in the first place.

**Comment by tomschenkjr** [comment]
@geneorama - Oh, and duh, I forgot that I [could have done it](https://github.com/blog/2224-change-the-base-branch-of-a-pull-request)

**Pull request merged**